sensor_msgs/test/test_pointcloud_conversion.cpp

```diff
@@ -18,7 +18,26 @@
 #include "sensor_msgs/msg/point_cloud.hpp"
 #include "sensor_msgs/msg/point_cloud2.hpp"
 
+#ifdef _MSC_VER
+#pragma warning(push)
+#else
+#pragma GCC diagnostic push
+#ifndef __clang__
+#pragma GCC diagnostic ignored "-Wcpp"
```

Comment:
I couldn't find the docs, is this for ignoring a `#pragma message`?

Reply:
If the answer is yes, you can ignore this other comment https://github.com/ros2/common_interfaces/pull/107#discussion_r409746682.

Reply:
yes, this is how to ignore the `#warning` in gcc, but apparently clang doesn't support this specific diagnostic. That's probably due to them having the same problem as g++ not being able to suppress it, but they at least don't silently fail to suppress it.